test.js

```diff
@@ -0,0 +1 @@
+(() => 'Hello World')();
```

Comment:
I would remove these files and copy the tests from my PR. The tests there worked. And in addition, I would enforce the CI to run them 👍

Reply:
Files removed! I added them by mistake